doc/newsfragments/2691_changed.fix_match_assertion.rst

```diff
@@ -0,0 +1 @@
+Introduced a new parameter ``include_only_tags_of_expected`` for py:meth:`result.fix.match() <testplan.testing.multitest.result.FixNamespace.match>` assertion, It will only compares the tags present in the expected message when the parameter is set to `True`.
```

Comment:
this is a bit long...suggest a few see if there's anything you like:

include_as_expected
include_per_expected
include_only_expected
include_tags_of_expected

Reply:
`inculude_only_expected` wins.